### DISCUSSION TITLE
Is it possible to install "redash" locally on m1 MacBook Pro?

### DISCUSSION THREAD

**Discussion by woongkiKim** [getredash/redash#6815] [Q&A]
Hello. I am a user using m1 MacBook Pro.

I have two questions.

1. Is it possible to install "redash" locally on m1 MacBook Pro?
2. I am currently using Docker with PostgreSQL on my local machine. Is it possible to connect to the installed redash?

![스크린샷 2024-03-19 오후 3 49 39](https://github.com/getredash/redash/assets/50348532/151f0c85-2362-483e-bde7-91f416002a6c)

2-1. An error message appeared on localhost:5001 of redash. I want to resolve the issue.

<img width="1502" alt="스크린샷 2024-03-19 오후 3 54 03" src="https://github.com/getredash/redash/assets/50348532/029351db-1dc1-4585-b2e7-9f5eacf78fb8">

**PS. I would really appreciate your help. This is my first time asking a question, so please forgive me if I come across as rude. 🙏 **

`
sqlalchemy.exc.InternalError: (psycopg2.errors.InFailedSqlTransaction) current transaction is aborted, commands ignored until end of transaction block

[SQL: SELECT organizations.updated_at AS organizations_updated_at, organizations.created_at AS organizations_created_at, organizations.id AS organizations_id, organ
izations.name AS organizations_name, organizations.slug AS organizations_slug, organizations.settings AS organizations_settings
FROM organizations
WHERE organizations.slug = %(slug_1)s
LIMIT %(param_1)s]
[parameters: {'slug_1': 'default', 'param_1': 1}]
(Background on this error at: http://sqlalche.me/e/13/2j85)
`

**Comment by gaecoli** [comment]
`docker-compose run server create_db`
You should run this line.

**Reply by woongkiKim** [reply]
Thank you for your answer. But it didn't work. 

So I built an environment on AWS with EC2 ubuntu and RDS. 

Is it possible to do this locally?


`redash git:(master) ✗ docker-compose run server create_db
[+] Running 2/0
 ⠿ Container redash-postgres-1  Running                  0.0s
 ⠿ Container redash-redis-1     Running                  0.0s
/usr/local/lib/python3.8/site-packages/qds_sdk/commands.py:1124: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if options.mode is "1":
/usr/local/lib/python3.8/site-packages/qds_sdk/commands.py:1137: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if options.db_update_mode is "updateonly":
/usr/local/lib/python3.8/site-packages/qds_sdk/commands.py:1524: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if (total is 0) or (downloaded == total):
[2024-03-22 08:01:24,023][PID:1][INFO][xmlschema] Resource 'XMLSchema.xsd' is already loaded`
